.github/workflows/scanner-versioned-definitions-update.yaml

```diff
@@ -310,15 +317,14 @@ jobs:
         grep -E "^[0-9]\.[0-9]+\.[0-9]+$" scanner/updater/version/RELEASE_VERSION | while read -r release; do
           case "$release" in
           4.4.*)
+            echo "Copy $release"
             gsutil cp "$single" "gs://definitions.stackrox.io/v4/vulnerability-bundles/$release/"
             ;;
-          4.5*)
+          4.5.*)
+            echo "Copy $release"
             gsutil cp "$single" "gs://definitions.stackrox.io/v4/vulnerability-bundles/$release/"
             gsutil cp "$multi" "gs://definitions.stackrox.io/v4/vulnerability-bundles/$release/"
             ;;
-          *)
```

Comment:
Based on https://github.com/stackrox/scanner/pull/1643 it seems like we plan on continuing to update this file in 4.6 and beyond, so this is no longer an error

Reply:
We still want to skip copying things for `>= 4.6`; so I think this check is valid, but yes, not an error.

Reply:
going to keep this stuff removed, then, since 4.6+ will just be ignored, then